### ISSUE TITLE
Add logout button

### ISSUE THREAD

**Issue by emilgoldsmith** [ChessCom/browser-extension#69]
As far as I can see there's currently no way to log out from within the extension, correct?

**Comment by martynchamberlin** [comment]
Correct, there currently is no logout button. Let me check with the team and get back with you on this.

**Comment by martynchamberlin** [comment]
@emilgoldsmith All right, let's build this if you wish! Where would you put the logout button?

**Comment by emilgoldsmith** [comment]
Good question... What about making the link to your own user-page (see red box below) a dropdown?

![logout button](https://cloud.githubusercontent.com/assets/18149242/23887125/d61bcaa0-084d-11e7-8ac4-fa029ba9cf6d.png)

as mentioned earlier I'm more of a back-end guy than a designer or strong front-end developer though I know some front-end. But again, as a user, I feel that it's intuitive that the image of me in the corner could be an options dropdown? We could also add a few more options in that case I guess, like "go to settings", "go to member page" or other links to different things that were related to you as a user.

What do Chess.com UX experts say to that?

**Comment by martynchamberlin** [comment]
Let's try to mimic the dropdown design at chess.com as much as possible. Here's a quick comp I threw together of what this could look like.

<img width="302" alt="screen shot 2017-03-14 at 4 17 02 pm" src="https://cloud.githubusercontent.com/assets/1827628/23922841/1007521e-08d2-11e7-83b3-61fe757a00ef.png">

**Comment by emilgoldsmith** [comment]
Yeah exactly, something like that would be great.

**Comment by nathanpdaniel** [comment]
@martynchamberlin (or anyone who knows the answer)... You wouldn't happen to know a good route for Log Out, would you?  `https://www.chess.com/logout` is not a good route from the extension (it puts a big red banner at the top and asks if the user "meant" to logout with a link to actually logout).

Once I can get that route, I'll create a PR which gives the above functionality.

I'll also do some more digging to see if I can't find the proper way to logout from the extension.

**Comment by martynchamberlin** [comment]
@nathanpdaniel The challenge here is that our logout is a POST and requires an internal CSR token that gets validated on the server. We'd have to build out an API method that allows the user to log out without this token. 

Cc @termleech
